src/measure_theory/measure_space.lean

```diff
@@ -1865,6 +1868,18 @@ instance add.sigma_finite (μ ν : measure α) [sigma_finite μ] [sigma_finite 
   sigma_finite (μ + ν) :=
 by { rw [← sum_cond], refine @sum.sigma_finite _ _ _ _ _ (bool.rec _ _); simpa }
 
+lemma sigma_finite_of_map (μ : measure α) {f : α → β} (hf : measurable f) [sigma_finite (map f μ)] :
```

Comment:
Do you really want to have `sigma_finite (map f μ)` as a typeclass parameter here? I don't expect that typeclass inference has instances that will help it show sigma-finiteness of this in any case, so I'd rather have it as a regular parameter.

I just noticed that this is the next lemma. So I'd just drop the first one, unless you have examples of applications.